src/Microsoft.AspNetCore.Server.Testing/Deployers/NanoDeployer/NanoDeployer.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Server.Testing
+{
+    public class NanoDeployer : ApplicationDeployer
```

Comment:
> Is there anything about this that is exclusive to Nano, or would this work against any remote Windows machine? RemoteDeployer?

Good question. There is nothing which is specific to Nano as such here. 

> This isn't plugged into the factory. How will it be used?
I initially created a ServerType of `Nano` but it felt out of place as other servers (except: nginx) are application servers like IIS, WebListener, Kestrel. I am (working on sending out a PR for MusicStore) just newing up this deployer instead of going via factory.